core/services/keystore/keys/ocr2key/ton_keyring.go

```diff
@@ -0,0 +1,122 @@
+package ocr2key
```

Comment:
I understand this PR is about following the existing structure, but what's the difference between this and https://github.com/smartcontractkit/chainlink/blob/9fbbf74e53af17d47e566503c551bd37512f3e88/core/services/keystore/keys/ocr2key/aptos_keyring.go?

This code is 1:1 duplicate, I there a specific reason we need it duplicated?

This is also relevant to tests, which I assume are also 1:1 duplicated.

----

Please follow up by scoping out a TON Hardening Epic, and adding few tasks to track this tech debt and come back to it eventually.

Reply:
Yes, implementation is almost 1:1 Aptos keyring, the only difference is the implementation of OCR3 Sign/Verify functions. I will create these issues to follow up 👍